Cargo.toml

```diff
@@ -15,6 +15,4 @@ keywords = ["storage"]
 categories = ["embedded", "hardware-support", "no-std"]
 
 [dependencies]
-nb = "1"
-no-std-net = "0.4"
 heapless = "^0.5"
```

Comment:
Can we bump heapless? The latest version (0.7) requires 1.51.0, though, due to const-generics.